gokart/gcs_obj_metadata_client.py

```diff
@@ -71,30 +81,63 @@ def add_task_state_labels(
             if update_response is None:
                 logger.error(f'failed to patch object {obj} in bucket {bucket} and object {obj}.')
 
+    @staticmethod
+    def _normalize_labels(labels: Optional[dict[str, str]]) -> dict[str, str]:
+        def _normalize_labels_helper(_params: Optional[dict[str, Any]]) -> dict[str, str]:
```

Comment:
[ask]
why do you create this inner function?
I feel this is enough for this purpose.

```python
    @staticmethod
    def _normalize_labels(labels: Optional[dict[str, str]]) -> dict[str, str]:
        return {str(key): str(value) for key, value in labels.items()} if labels else {}
```